Makefile

```diff
@@ -123,21 +123,21 @@ install: install-server install-ui ## Build and install
 watch: ## Watch packages. For use alongside jupyter lab --watch
 	export PATH=$$(pwd)/node_modules/.bin:$$PATH && lerna run watch --parallel
 
-test-server: install-server ## Run unit tests
+test-server: install-server # Run unit tests
 	pytest -v elyra
 
-test-ui: lint-ui test-ui-unit test-ui-integration ## Run frontend tests
+test-ui: lint-ui test-ui-unit test-ui-integration # Run frontend tests
 
-test-ui-integration: ## Run frontend cypress integration tests
+test-ui-integration: # Run frontend cypress integration tests
 	npm run test:integration
 
-test-ui-unit: ## Run frontend jest unit tests
+test-ui-unit: # Run frontend jest unit tests
 	npm run test:unit
 
 test-ui-debug: ## Open cypress integration test debugger
```

Comment:
This target seems out of place in this new context of only displaying top-level targets.

Reply:
I thought about that, but thought this one was good for users to be aware of as it would launch the cypress debugger to help identify issues. But I agree that it feels a little out of place. If you prefer I can hide it.

Reply:
If we're going to be consistent, it needs to be hidden.  There are plenty of targets that would be good for users performing specific actions to be aware of and they'll just need to peruse the Makefile to discover them.

Target `validate-runtime-images` should also probably be removed from the display since its performed automatically by CI and would only produce different results when the factory runtime-images are updated (which is rare).

Reply:
The part of this that bothers me a bit is that the target is called `test-ui-debug`, but it only runs the integration tests. Shouldn't we call this `test-integration-debug`?

Reply:
`test-integration-debug` should be hidden and can be documented elsewhere, just like all the other useful (but hidden) targets.